docs: update doc for guide user to config auth etcd (#8849)
apache/apisix: docs/en/latest/FAQ.md

    prefix: "/apisix"
```

> ## How does APISIX configure ETCD with authentication

Reviewer: ```suggestion
## How does APISIX configure ETCD with authentication?
```

> If you are using an auth ETCD instance, you need to use the correct user and password to access it. You need to do edit this in the `conf/config.yaml` file.

Reviewer 2: ```suggestion
Suppose you have an ETCD cluster that enables the auth. To access this cluster, you need to configure the correct username and password for Apache APISIX. Edit the `conf/config.yaml` file as follows:
```

> Suppose you have an ETCD cluster that enables the auth. To access this cluster, you need to configure the correct username and password for Apache APISIX. Edit the `conf/config.yaml` file as follows:

Reviewer 2: ```suggestion
Suppose you have an ETCD cluster that enables the auth. To access this cluster, you need to configure the correct username and password for Apache APISIX in `conf/config.yaml`:
```

```yaml
deployment:
  etcd:
    host:
      - "http://127.0.0.1:2379"

>     user: root                     # root username for etcd
>     password: 5tHkHhYkjr6cQY       # root password for etcd

Reviewer 3: ```suggestion
    user: etcd_user                       # root username for etcd
    password: etcd_password     # root password for etcd
```
Would be better to don‘t have a specific password, Misunderstandings may arise.

>     user: etcd_user                       # root username for etcd
>     password: etcd_password     # root password for etcd

Reviewer 3: ```suggestion
    user: etcd_user             # username for etcd
    password: etcd_password     # password for etcd
```
It is __not recommended__ to use `root` as etcd user.

```

> For other ETCD configurations, such as expiration times, retries, and so on, you can see the `ETCD` section in the `conf/config-default.yaml` file.

Reviewer 3: ```suggestion
For other ETCD configurations, such as expiration times, retries, and so on, you can refer to the `ETCD` section in the `conf/config-default.yaml` file.
```
